web_src/js/modules/tippy.js

```diff
@@ -6,7 +6,7 @@ export function createTippy(target, opts = {}) {
     placement: target.getAttribute('data-placement') || 'top-start',
     animation: false,
     allowHTML: false,
-    hideOnClick: false,
+    hideOnClick: true,
```

Comment:
If you ever blamed the history, you would see that it's a whack-a-mole game.

You need some Global Framework Thinking and Design Patterns to resolve it.

Reply:
Done.